code/modules/projectiles/guns/projectile/sniper.dm

```diff
@@ -65,3 +67,40 @@
 
 	toggle_scope(2.0)
 
+////////////// Dragunov Sniper Rifle //////////////
+
+/obj/item/weapon/gun/projectile/SVD
+	name = "\improper Dragunov"
+	desc = "The SVD, also known as the Dragunov, was mass produced with an Optical Sniper Sight so simple that even Ivan can figure out how it works. Too bad for you that it's written in Russian. Uses 7.62mm rounds."
```

Comment:
Still, I don't understand the problem with pop culture references. This is really the most subtle one here.